src/edu/psu/sweng500/team8/play/SudokuAction.java

```diff
@@ -14,12 +15,14 @@
 	private CellCoordinates cellCordinates;
 	private int previousValue;
 	private int newValue;
+	private PencilMarkManager pencilMarkManager; 
 
 	public SudokuAction(CellCoordinates cellCoordinates,

```

Comment:
Need more info on "Might be easier to just keep the Cell itself"
I will respond to other emails once I get home.
 On Jun 18, 2015 5:58 PM, "Jeremy Nelson" <notifications@github.com> wrote:

> In src/edu/psu/sweng500/team8/play/SudokuAction.java
> <https://github.com/garnryang/Team8/pull/3#discussion_r32783512>:
>
> > @@ -14,12 +15,14 @@
> >  	private CellCoordinates cellCordinates;
> >  	private int previousValue;
> >  	private int newValue;
> > +	private PencilMarkManager pencilMarkManager;
> >
> >  	public SudokuAction(CellCoordinates cellCoordinates,
>
> Might be easier to just keep the Cell itself
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garnryang/Team8/pull/3/files#r32783512>.
>